R/Archive.R

```diff
@@ -43,17 +43,16 @@ Archive = R6Class("Archive",
     #'
     #' @param xss_trafoed (`list()`).
     add_evals = function(xdt, xss_trafoed, ydt) {
-
-      # FIXME: add checks here for the dts and their domains
-      # FIXME: make asserts better!
       assert_data_table(xdt)
       assert_data_table(ydt)
       assert_list(xss_trafoed)
+      map(ydt[, self$cols_y, with = FALSE], function(y) {
```

Comment:
Can you please use `check_data_table(..., any.missing = FALSE)` instead?

Reply:
Thanks.I did not know about this. But we want to use `assert_data_table`, right?